test/Reflection/ReflectionClassTest.php

```diff
@@ -18,6 +18,12 @@ private function getComposerLocator()
         return new ComposerSourceLocator($loader);
     }
 
+    /**
+     * @covers \BetterReflection\Reflection\ReflectionClass::inNamespace()
```

Comment:
I suggest avoiding this sort of strict coverage and defining coverage only at class level: simpler/easier